src/blas/backends/armpl/CMakeLists.txt

```diff
@@ -0,0 +1,78 @@
+#===============================================================================
+# Copyright 2024 SiPearl
+# Copyright 2020-2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+#
+# SPDX-License-Identifier: Apache-2.0
+#===============================================================================
+
+set(LIB_NAME onemath_blas_armpl)
+set(LIB_OBJ ${LIB_NAME}_obj)
+
+# Add third-party library
+find_package(ARMPL REQUIRED)
+
+set(SOURCES armpl_common.hpp
+  armpl_level1.cpp armpl_level2.cpp armpl_level3.cpp armpl_batch.cpp armpl_extensions.cpp
+  $<$<BOOL:${BUILD_SHARED_LIBS}>: armpl_wrappers.cpp>
+)
+add_library(${LIB_NAME})
+add_library(${LIB_OBJ} OBJECT ${SOURCES})
+add_dependencies(onemath_backend_libs_blas ${LIB_NAME})
+
+if (USE_ADD_SYCL_TO_TARGET_INTEGRATION)
+  add_sycl_to_target(TARGET ${LIB_OBJ} SOURCES ${SOURCES})
+endif()
+
+target_include_directories(${LIB_OBJ}
+  PRIVATE ${PROJECT_SOURCE_DIR}/include
```

Comment:
I would suggest to make some small changes to this CMake to match with this PR: https://github.com/uxlfoundation/oneMath/pull/627/files#diff-57a0c18793f49b94598346caa9e5e0844e9c218cd310d4b2846865714bce5d5c
This should help to install oneMath with the arm backend and use it in external project.

Reply:
I tested it locally and it does not seem to harm, so I'm fine jumping te gun and doing the change before the PR is actually merged. Thanks

Reply:
Done